tools/memap.py

```diff
@@ -8,26 +8,31 @@
 import csv
 import json
 import argparse
+from copy import deepcopy
 from prettytable import PrettyTable
 
 from utils import argparse_filestring_type, \
     argparse_lowercase_hyphen_type, argparse_uppercase_type
 
-DEBUG = False
-
 RE_ARMCC = re.compile(
     r'^\s+0x(\w{8})\s+0x(\w{8})\s+(\w+)\s+(\w+)\s+(\d+)\s+[*]?.+\s+(.+)$')
 RE_IAR = re.compile(
     r'^\s+(.+)\s+(zero|const|ro code|inited|uninit)\s'
     r'+0x(\w{8})\s+0x(\w+)\s+(.+)\s.+$')
 
+# Pending:
+# 1. Add tests
+# 2. Update markdown documentation
+
+
 class MemapParser(object):
     """An object that represents parsed results, parses the memory map files,
     and writes out different file types of memory results
     """
 
     print_sections = ('.text', '.data', '.bss')
 
+    # Info from this section can be shown as misc or startup code (.text)
```

Comment:
Is this comment really needed? I think that the variable is named well enough to get the point across.

This will not block the PR